README.md

```diff
@@ -216,17 +201,17 @@ export const Todo = {
     due_date: "DATE",
     importance: "INTEGER",
   },
-  belongsTo: [{ target: "User", options: { as: "user" } }], // 👀
+  belongsTo: [{ target: "User", options: { as: "user" } }],
 }
 ```
 
-You can find all of the possible [datatypes
-here](https://sequelize.org/docs/v7/other-topics/other-data-types).
+You can find all of the possible data types for a schema's `attributes`
+[here](https://sequelize.org/docs/v7/other-topics/other-data-types).
```

Comment:
This link 404s. I wasn't able to find an obvious replacement for it, so I'm not sure where it's supposed to go

Reply:
@luca-bitovi https://sequelize.org/docs/v6/other-topics/other-data-types/ would suffice.

Reply:
Done!